addon/components/offering-form.hbs

```diff
@@ -244,7 +244,7 @@
           </p>
         {{/each}}
       </div>
-      <div class="buttons">
+      <div class="buttons" {{ref this "scrollTo"}}>
```

Comment:
`{{ref}}` is currently bound to the ember runloop and sometimes these references aren't set when expected. I'd suggest adding a specific `{{did-insert}}` directly here that way you're sure you get the element reference and you can still check the `@scrollToBottom` value.

Reply:
i don't see a way to chain `did-insert` modifiers. and i don't think they stack in order specified. or do they?

most of the markup in this  component gets rendered after the `loadData` task has completed, including the buttons container that we want to scroll to. so whatever code invokes scroll-into-view will have to run afterwards.
that's why i'm running this sequentially within the `load` task.